tensorflow/core/kernels/image/adjust_contrast_op.cc

```diff
@@ -64,6 +65,14 @@ class AdjustContrastOp : public OpKernel {
                 errors::InvalidArgument("max_value must be scalar: ",
                                         max_value.shape().DebugString()));
 
+    if (std::is_same<Device, GPUDevice>::value) {
+      OP_REQUIRES(
+          context, !OpDeterminismRequired(),
+          errors::Unimplemented(
+              "A deterministic GPU implementation of AdjustContrastOp"
```

Comment:
Change AdjustContrastOp to AdjustContrast, since AdjustContrast is the public op name. And in the error below, change AdjustContrastOpv2 to AdjustContrastv2

Reply:
Wonderful. Thanks. I now have more clarity about what name to use in future exceptions.